src/backend/distributed/utils/ruleutils_10.c

```diff
@@ -0,0 +1,7745 @@
+/*-------------------------------------------------------------------------
+ *
+ * ruleutils.c
+ *	  Functions to convert stored expressions/querytrees back to
+ *	  source text
+ *
+ * Portions Copyright (c) 1996-2017, PostgreSQL Global Development Group
+ * Portions Copyright (c) 1994, Regents of the University of California
+ *
+ *
+ * IDENTIFICATION
+ *	  src/backend/distributed/utils/ruleutils_96.c
```

Comment:
This is a typo.

I copied the file from PostgreSQL 10 then applied the Citus changes (the commit history should reflect this).

I'll bet I just region-copied the whole Citus header and forgot to modify the version in the filename (so no worries that I accidentally e.g. copied 9.6 and called it 10).